### DISCUSSION TITLE
Not able to see the GitHub Stats cards

### DISCUSSION THREAD

**Discussion by yaten2302** [anuraghazra/github-readme-stats#3108] [Q&A] [answered]
### Describe the Bug

Hi,

When I'm running the ```vercel dev``` command, I'm getting redirected to this page.

![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/b264fde0-e59b-45c3-850d-6b43993fb167)

![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/74bb3708-bc65-4afd-bab8-3c753e8106bf)

**Comment by rickstaa** [comment]
@yaten2302 Amazing that you are trying to run the project locally and consider contributing to our project 🚀. The behaviour you are experiencing is the one you want. It is caused by the following lines in the `vercel.json` file:

https://github.com/anuraghazra/github-readme-stats/blob/03b0adc8b73904efc8c4600a7330b6c2803b78f8/vercel.json#L8-L13

If you want to test cards locally, like the remote version, you must supply it with query parameters to work (i.e. `localhost:3000/api?username=anuraghazra). If you are considering contributing, you can also check out the [CONTRIBUTING.md](https://github.com/anuraghazra/github-readme-stats/blob/master/CONTRIBUTING.md) file for more information.

**Reply by yaten2302** [reply]
So, like in the ```vercel.json``` file, do I've to change "destination": "https://github.com/anuraghazra/github-readme-stats"
to
"destination": "localhost:3000/api?username=yaten2302" ?

**Reply by yaten2302** [reply]
Also, I want to contribute to this project, but I've never worked on any advanced projects. So, in case of any problem, should I open a discussion here?

**Comment by rickstaa** [comment, marked as answer]
> So, like in the `vercel.json` file, do I've to change "destination": "https://github.com/anuraghazra/github-readme-stats" to "destination": "localhost:3000/api?username=yaten2302" ?

No, sorry for being unclear. You can use that URL in the browser. You don't need to make any changes to the Vercel.json. 

If you run into any problems contributing, feel free to open an issue, and me or the other collaborators will respond to help you 👍🏻.

**Reply by yaten2302** [reply]
Now it's working, Thank you so much 😊, will surely try to contribute to this wonderful project 👍

**Reply by rickstaa** [reply]
> Now it's working, Thank you so much 😊, will surely try to contribute to this wonderful project 👍

Feel free to tag me if you have any questions or run into other problems. Our top feature requests, bugs etc., can be found at https://github.com/anuraghazra/github-readme-stats/issues/1935. If you want to take something on, you [can create an issue](https://github.com/anuraghazra/github-readme-stats/issues/new/choose) or ask to be assigned to [an existing issue](https://github.com/anuraghazra/github-readme-stats/issues). We will also participate again in this year's https://hacktoberfest.com/.
